README.md

```diff
@@ -154,6 +143,12 @@ $ fuseml COMMAND --help
 
 ```
 
+### Usage
+
+After using `fuseml-installer` to install required components of FuseML, for actual work with FuseML one needs to use `fuseml-core` command line client.
+
+Go to [fuseml-core project page](https://github.com/fuseml/fuseml-core) to learn about CLI.
+
 ## Configuration
```

Comment:
Is that section still valid?

Reply:
I planned to look into that. I believe it still has its use, or that we can reuse it for something else. If I find out the opposite, I'll remove it as well.